app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +28,12 @@
             </intent-filter>
         </receiver>
 
+        <service
```

Comment:
Why not simply send intents to MainService?

Reply:
Possibly due to "Separation of Concerns" and "Single Responsibility Principle". Otherwise no reason.

Reply:
My idea is to have the MainService as the single point of responsibility that takes its commands via Intents so that:
* external apps can control the MainService
* the MainActivity aka Admin Panel should use this as well, i.e. the static methods should be replaced by Intents so that the only interface to the MainService are Intents